account_reconcile_oca/static/src/js/reconcile_form/reconcile_form_controller.esm.js

```diff
@@ -23,9 +23,11 @@ export class ReconcileFormController extends FormController {
     async reloadFormController() {
         var is_reconciled = this.model.root.data.is_reconciled;
         await this.model.root.load();
-        if (!is_reconciled && this.model.root.data.is_reconciled) {
-            // This only happens when we press the reconcile button
-            if (this.env.parentController) {
+        // This only happens when we press the reconcile button
```

Comment:
Is this related to the commit message?

Reply:
Yes it is. If we don't apply this change, the update is only done when  we go from not reconciled to reconciled

Reply:
But then the comment seems incorrect, as this happens a lot of times...

Reply:
True, I just moved the comment to the right place (next if) and put the right comment there. Thanks